### ISSUE TITLE
Wifi connection randomly(?) lost

### ISSUE THREAD

**Issue by flo269** [vitotai/BrewPiLess#137]
Hi,

I'm running v3.2 and during my current fermentation I am experiencing wifi connectivity issues. Brewpiless seems to lose connection to wifi: This happened twice in the last few days (see flat filtered SG graphs). I then pulled the plug to reset and everything worked fine again.

What could I do to deliver more precise info?

Cheers
Flo

![screenshot 2018-10-21 at 11 58 41](https://user-images.githubusercontent.com/25147331/47265501-21aa8980-d529-11e8-93a2-860efda38bc5.png)

**Comment by flo269** [comment]
@amlynarczyk do you mean the bin file with the new ui or when you install the new ui manually?

I'm not sure if its a ui issue because I can't reach the ui in first place and I can't reach BPL via ping either.

**Comment by amlynarczyk** [comment]
Bin file

**Comment by vitotai** [comment]
I had the same issue. I've followed this issue for a while without luck.

The new ui version uses framework 1.8.0 with latest ESPAsyncWebServer library, while the classic one uses 1.5.0 with older version of ESPAsyncWebServer library.

I've tried classic ui with 1.8.0, but it was more unstable.

One possible issue I've found is trying to reconnect if BPL found it disconnected. I am trying to change it by letting the WiFi library do the job instead of doing it manually. 
The code will be available soon if the test platform doesn't crash for at least 24 hours.
